charactersheet/charactersheet/index.html

```diff
@@ -73,100 +73,131 @@ <h2 data-bind='text: playerTitle'></h2>
+		<div class="row">
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div data-bind="template: { name: 'feats_prof.tmpl', templateUrl: 'feats_prof', data: featsProfViewModel }">
 				</div>
-				<div class="col-md-8 col-sm-6 col-padded">
-					<div id="equipment" data-bind="template: { name: 'equipment.tmpl', templateUrl: 'equipment', data: equipmentViewModel }">
-					</div>
-					<div id="spells" data-bind="template: { name: 'spells.tmpl', templateUrl: 'spells', data: spellbookViewModel }">
-					</div>
-					<div data-bind="template: { name: 'treasure.tmpl', templateUrl: 'treasure', data: treasureViewModel }">
-					</div>
+			</div>
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="skills" data-bind="template: { name: 'skills.tmpl', templateUrl: 'skills', data: skillsViewModel }">
 				</div>
 			</div>
-			<div class="row">
-				<div class="col-sm-6 col-padded">
-					<div id="weapons" data-bind="template: { name: 'weapons.tmpl', templateUrl: 'weapons', data: weaponsViewModel }">
-					</div>
+		</div>
+	</div>
+<!-- End Skills Tab -->
+	<div role="tabpanel" class="tab-pane" id="spells" data-bind="with: spellsTabViewModel, css: spellsTabStatus">
+		<div class="row">
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="spells" data-bind="template: { name: 'spells.tmpl', templateUrl: 'spells', data: spellbookViewModel }">
 				</div>
-				<div class="col-sm-6 col-padded">
-					<div id="armors" data-bind="template: { name: 'armors.tmpl', templateUrl: 'armor', data: armorViewModel }">
-					</div>
 			</div>
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="spell_slots" data-bind="template: { name: 'spell_slots.tmpl', templateUrl: 'spell_slots', data: spellSlotsViewModel }">
+				</div>
 			</div>
-			<div class="row">
-				<div class="col-sm-12 col-padded">
-					<div data-bind="template: { name: 'notes.tmpl', templateUrl: 'notes', data: notesViewModel }">
-					</div>
+		</div>
+	</div>
+<!-- End Spells Tab -->
+	<div role="tabpanel" class="tab-pane" id="equipment" data-bind="with: equipmentTabViewModel, css: equipmentTabStatus">
+		<div class="row">
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="weapons" data-bind="template: { name: 'weapons.tmpl', templateUrl: 'weapons', data: weaponsViewModel }">
+				</div>
+			</div>
+			<div class="col-md-6 col-sm-12 col-padded">
+				<div id="armors" data-bind="template: { name: 'armors.tmpl', templateUrl: 'armor', data: armorViewModel }">
 				</div>
 			</div>
 		</div>
-<!-- End Character Tab -->
-		<div role="tabpanel" class="tab-pane" id="dm" data-bind="with: dmTabViewModel, css: dMTabStatus">
-			<div data-bind="template: { name: 'campaign.tmpl', templateUrl: 'campaign', data: campaignViewModel }">
+	</div>
+<!-- End Equipment Tab -->
+	<div role="tabpanel" class="tab-pane" id="inventory" data-bind="with: inventoryTabViewModel, css: inventoryTabStatus">
```

Comment:
This is the last pane. Its has the same issue as the spells.

![screenshot 2016-02-03 09 58 26](https://cloud.githubusercontent.com/assets/3310280/12791822/a9738bde-ca5d-11e5-83aa-b0d00e91160a.png)